### DISCUSSION TITLE
Angular dashboard gets empty data from query-builder

### DISCUSSION THREAD

**Discussion by GitVivekHub** [Sunbird-cQube/community#784] [General]
What microservices are needed to be started when ingesting data locally?So far im able to run dashboard-ms,ingestion-ms and query-builder-ms. By doing so im able to upload the file but somehow my angular app is not able to get the processed data from the query-builder.Can anyone help me with this. Im running the app locally

**Comment by RNarmatha** [comment]
Hi @GitVivekHub - Kindly ensure that  postgres has  installed.  
Before data to be visualized into dashboard make sure  check dataset are in the correct format
